akka-actor-testkit-typed/src/test/java/akka/actor/testkit/typed/javadsl/ActorTestKitTest.java

```diff
@@ -51,10 +50,12 @@ public void systemNameShouldComeFromGivenClassName() {
   @Test
   public void testKitShouldSpawnActor() throws Exception {
     final CompletableFuture<Done> started = new CompletableFuture<>();
-    testKit.spawn(Behaviors.setup((context) -> {
-      started.complete(done());
-      return Behaviors.same();
-    }));
+    testKit.spawn(
+        Behaviors.setup(
+            (context) -> {
+              started.complete(done());
+              return Behaviors.same();
+            }));
```

Comment:
We don't want 4 space indentation, even if that is the "normal" for Java, it will break lots of documentation pages by making the code too wide. I also think it does to much line breaking making some stuff below very verbose/long/harder to read. Are there some options to configure the sbt-java-formatter?

Reply:
No configuration, but could be fallback to 0.3.1 which is using the eclipse one.